.ci/scripts/publish_docs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# WARNING: DO NOT EDIT!
+#
+# This file was generated by plugin_template, and is managed by it. Please use
+# './plugin-template --github pulpcore' to update this file.
+#
+# For more info visit https://github.com/pulp/plugin_template
+
+set -euv
+
+# make sure this script runs at the repo root
+cd "$(dirname "$(realpath -e "$0")")"/../..
+
+mkdir ~/.ssh
+echo "$PULP_DOCS_KEY" > ~/.ssh/pulp-infra
+chmod 600 ~/.ssh/pulp-infra
+
+echo "docs.pulpproject.org,8.43.85.236 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBGXG+8vjSQvnAkq33i0XWgpSrbco3rRqNZr0SfVeiqFI7RN/VznwXMioDDhc+hQtgVhd6TYBOrV07IMcKj+FAzg=" >> /home/runner/.ssh/known_hosts
```

Comment:
```suggestion
echo "docs.pulpproject.org,8.43.85.236 ecdsa-sha2-nistp256 [newKey]" >> /home/runner/.ssh/known_hosts
```
this is the key from pulp-operator, need to create one for this repo

Reply:
https://github.com/pulp/pulp-infra-ansible/pull/27

Reply:
actually this is the plugin-template key, I was thinking it was pulp-operator's key (I copied the script from there)